iofog-agent-daemon/src/main/java/org/eclipse/iofog/field_agent/FieldAgent.java

```diff
@@ -1378,7 +1378,9 @@ public void instanceConfigUpdated() {
         } catch (Exception e) {
             logError("Error posting updated for config ", e);
         }
-        orchestrator.update();
+        if (orchestrator != null) {
+            orchestrator.update();
+        }
```

Comment:
And what happens if `orchestrator` is null? Can it ever be null?

Reply:
No it should not be null ever.